modules/profile/public/lib/queries/profiles/find_proxy.liquid

```diff
@@ -1,6 +1,9 @@
 {% liquid
   function profile_module = 'modules/core/queries/variable/get', name: 'PROFILE_MODULE', default: 'modules/profile'
   assign function_name = profile_module | append: '/queries/profiles/find'
-  function profile = function_name, user_id: user_id, id: id, slug: slug, uuid: uuid, filters: filters
+if slug
+ assign user_id =slug[2]
+endif
+  function profile = function_name, user_id: user_id, id: id, first_name: slug[0] , last_name:slug[1] , uuid: uuid, filters: filters
```

Comment:
we shouldn't allow to find a user based on first name and last name - there will be a lot of John Smith